src/obu.c

```diff
@@ -378,10 +382,15 @@ static avifBool parseAV2SequenceHeader(avifBits * bits, avifSequenceHeader * hea
 
     avifBitsRead(bits, 2);       // enable_superres, enable_cdef
     if (avifBitsRead(bits, 1)) { // enable_restoration
-#if CONFIG_LR_FLEX_SYNTAX
-        avifBitsRead(bits, 2 + CONFIG_PC_WIENER + CONFIG_WIENER_NONSEP); // lr_tools_disable_mask[0]
-        if (avifBitsRead(bits, 1)) {                                     // uv_neq_y
-            avifBitsRead(bits, 2 + CONFIG_WIENER_NONSEP);                // lr_tools_disable_mask[1]
+#if CONFIG_LR_IMPROVEMENTS
+        avifBitsRead(bits, RESTORE_SWITCHABLE_TYPES);
+        if (avifBitsRead(bits, 1)) {
```

Comment:
Please add the original AVM variable name as a comment for future reference

Reply:
Done